src/styles/views/packages/styles.less

```diff
@@ -9,6 +9,25 @@
     }
   }
 
+  .filter-bar-item {
+
+    .button-outline {
+
+      span {
+        margin: @base-spacing-unit * 1/6;
+        padding: 1px 6px;
```

Comment:
I know @mperrotti said not to change the default styles, but without this the badges look ugly, since they are getting this rule
```
button > :last-child, .button > :last-child {
    padding-right: 0;
```
![Screenshot from 2019-08-20 15-23-31](https://user-images.githubusercontent.com/40791275/63346910-a723e380-c35e-11e9-8764-a30a64616128.png)